Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -340,6 +352,153 @@ instance uniqueOneLeftMoves : Unique (LeftMoves 1) :=
 instance isEmpty_one_rightMoves : IsEmpty (RightMoves 1) :=
   instIsEmptyPEmpty
 
+/-! ### Identity -/
+
+/-- Two pre-games are identical if their left and right sets are identical.
+That is, `Identical x y` if every left move of `x` is identical to some left move of `y`,
+every right move of `x` is identical to some right move of `y`, and vice versa. -/
+def Identical : ∀ (_ _ : PGame), Prop
```

Comment:
```suggestion
def Identical : ∀ (_ _ : PGame.{u}), Prop
```
We don't ever want to compare two `PGame`s from different universes, right ?

Reply:
Maybe we might want to do this, but we shouldn't do it anyways. The correct way would be to introduce `PGame.lift`, analogously to e.g. ordinals and cardinals.